Dockerfile

```diff
@@ -1,8 +1,16 @@
 FROM python:3.12.1
+
 ENV PYTHONBUFFERED 1
 ENV PYTHONDONTWRITEBYTECODE 1
+
 RUN mkdir /code
 WORKDIR /code
-COPY requirements.txt /code/
-RUN pip install --upgrade pip && pip install -r requirements.txt
-COPY . /code/
\ No newline at end of file
+COPY poetry.lock /code/
+COPY pyproject.toml /code/
+
+RUN pip install poetry
```

Comment:
use specific version

Reply:
@szmktk this some like "procedure" We need to know what we use(or will use); I know that tool from fast api, but not only me participate in the Team Zeton that I write about discuss about this good feature. I recommend @szmktk   idea 👍 to adapt poetry in Zeton 💯